paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerAdapter.kt

```diff
@@ -7,11 +7,11 @@ import com.stripe.android.model.PaymentMethod
 import com.stripe.android.paymentsheet.model.SavedSelection
 
 /**
- * [CustomerAdapter] A bridge to your backend to fetch Customer-related information. Typically,
```

Comment:
These changes seem unrelated. Ideally they'd be separate!

Reply:
I saw some typos and I fixed some typos 🤷‍♂️